src/components/Actions/Actions.css

```diff
@@ -5,22 +5,15 @@
 
   display: flex;
   flex-wrap: wrap;
-  justify-content: space-around;
```

Comment:
`ul.scores` will not be centered horizontally without this line in iPad portrait mode.

Reply:
I overlooked this, it's fixed now.